### ISSUE TITLE
Clarify the installation of vampire in the manual

### ISSUE THREAD

**Issue by teiesti** [potassco/anthem#246]
In order to improve accessibility for users, it is probably wise to clarify in the manual (https://potassco.org/anthem/installation.html) how vampire should be installed.

@javier-romero
